assets/i18n/id.json

```diff
@@ -25,245 +24,238 @@
-    "dgc-windows-wallet-modal-dogecoin-core-pro": "Tetapi ini adalah keuntungan karena aman untuk menambang Dogecoin dengan dompet ini, dompet ini dapat menangani transaksi dari pembayaran. Ini juga akan membantu menjaga jaringan Dogecoin.", 
+    "dgc-windows-wallet-modal-different-strokes": "Kedua dompet tersebut punya kelebihan dan kekurangan masing-masing.", 
+    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge itu dompet yang sangat ringan, jadi hanya \"memindai\" dalam blockchain sebelum menjadi tidak berguna lagi. Proses ini dilakukan sangat cepat.", 
+    "dgc-windows-wallet-modal-multidoge-con": "Tapi dompet ini hanya digunakan untuk penggunaan yang wajar saja. Dompet tersebut <strong>tidak bisa</strong> digunakan untuk menambang Dogecoin, karena akan sangat lambat dan tidak responsif.", 
+    "dgc-windows-wallet-modal-dogecoin-core-con": "Sementara, Dogecoin Core mendownload keseluruhan data blockchain. Jadi, sinkronisasi awal agak lambat dibandingkan MultiDoge, dan memakan banyak ruang penyimpanan.", 
+    "dgc-windows-wallet-modal-dogecoin-core-pro": "Namun tentunya ada keuntungannya. Dogecoin jadi lebih aman dengan dompet ini, selain bisa melakukan transaksi, dompet ini juga menjaga jaringan Dogecoin tetap gesit.", 
     "dgc-osx-wallet-modal-dogecoin-core": "Dogecoin Core.", 
-	"dgc-android-wallet-modal-title": "UNDUH MELALUI GOOGLE PLAY.",
+	"dgc-android-wallet-modal-title": "UNDUH DARI GOOGLE PLAY.",
     "dgc-android-wallet-modal-apk": "<a href=\"https://github.com/langerhans/dogecoin-wallet-new/releases/download/1.08/wallet-1.08-aligned.apk\">Direct APK Download.</a>",
-	"dgc-ios-wallet-modal-title": "DOWNLOAD MELALUI THE APP STORE.",
+	"dgc-ios-wallet-modal-title": "UNDUH DARI APP STORE.",
     "dgc-getting-started-title": "MEMULAI",
-    "dgc-getting-started-desc": "MEMULAI DENGAN DOGECOIN",
-    "dgc-getting-started-is-123": "adalah <strong>semudah</strong> 1, 2, 3!",
+    "dgc-getting-started-desc": "Mulai menggunakan Dogecoin",
+    "dgc-getting-started-is-123": "itu <strong>Gampang Banget</strong>!",
     "dgc-getting-started-desktop": "Dompet Desktop",
-    "dgc-getting-started-phone": "Dompet Phone",
-    "dgc-getting-started-online": "Dompet Online",
-    "dgc-getting-started-paper": "Dompet Paper",
+    "dgc-getting-started-phone": "Dompet Ponsel",
+    "dgc-getting-started-online": "Dompet Daring",
+    "dgc-getting-started-paper": "Dompet Kertas",
     "dgc-getting-started-choose": "Pilih jenis dompet untuk memulai.",
-    "dgc-getting-started-desktop-os": "Silahkan pilih sistem operasi komputermu.",
-    "dgc-getting-started-phone-os": "Silahkan pilih sistem operasi smartphonekmu.",
-    "dgc-getting-started-online-choice": "Silahkan pilih dompet onlinemu.",
+    "dgc-getting-started-desktop-os": "Pilih Sistem Operasi Komputer Kamu.",
+    "dgc-getting-started-phone-os": "Pilih Sistem Operasi Ponsel-Pintar Kamu.",
```

Comment:
It should be `Ponsel Pintar` without hypen, i think?

https://kbbi.kemdikbud.go.id/entri/ponsel%20pintar